### PR TITLE
PICT Tech Wizards

### PR DESCRIPTION
**Pull request by FireFeast7**
Code Submission of Tech Wizards for Web Weaver at Credenz'24

### CODE REVIEW

**Review by gautamgambhir97 — COMMENTED** [review]
Contributing
https://github.com/fetchai/uAgents/blob/main/integrations/CONTRIBUTING.md

Include a project.json file, example of:

```
{
    "title": "Bert Base Uncased",
    "description": "BERT base model (uncased) Pretrained model on English language using a masked language modeling (MLM) objective.",
    "categories":    ["Text Classification", "Hugging Face", "Text Generation"], 
    "deltav": false
}
```
Set deltaV to true if your agent is built using agentverse and is accessible by DeltaV.

File: integrations/career-craft/src/agents/resume creation/agent.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    await ctx.send(
+        sender, UAgentResponse(message=response, type=UAgentResponseType.FINAL)
+    )
+
+
+agent.include(simples)
+
+
+# async def handle_query_request(ctx: Context, sender: str, msg: BookEventRequest):
+#     ctx.logger.info(f"Received message from {sender}, session: {ctx.session}")
```
Comment (gautamgambhir97) [inline integrations/career-craft/src/agents/resume creation/agent.py:76]: Remove commented-out code

File: integrations/career-craft/src/agents/resume creation/agent.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# class UserProfile(BaseModel):
+    # name: str = Field(description="Full name of the person.")
+    # email: str = Field(description="Email address of the person.")
+    # phone: str = Field(description="Contact phone number.")
+    # address: str = Field(description="Physical address of the person.")
+    # skills: str = Field(description="Comma-separated list of skills the person has acquired.")
+    # certifications: str = Field(description="Comma-separated list of certifications the person holds.")
+    # awards: str = Field(description="Comma-separated list of awards the person has received.")
+    # experiences: str = Field(description="Comma-separated list of professional experiences.")
+
```
Comment (gautamgambhir97) [inline integrations/career-craft/src/agents/resume creation/agent.py:29]: Remove commented-out code

File: integrations/career-craft/src/agents/resume and job matching/agent.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    await ctx.send(
+        sender, UAgentResponse(message=response['response'], type=UAgentResponseType.FINAL)
+    )
+
+
+agent.include(simples)
+
+
+# async def handle_query_request(ctx: Context, sender: str, msg: BookEventRequest):
+#     ctx.logger.info(f"Received message from {sender}, session: {ctx.session}")
```
Comment (gautamgambhir97) [inline integrations/career-craft/src/agents/resume and job matching/agent.py:46]: Remove commented-out code

File: integrations/career-craft/src/agents/cold mailing/agent.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    await ctx.send(
+        sender, UAgentResponse(message="Email Sent Successfully", type=UAgentResponseType.FINAL)
+    )
+
+
+agent.include(simples)
+
+
+# async def handle_query_request(ctx: Context, sender: str, msg: BookEventRequest):
+#     ctx.logger.info(f"Received message from {sender}, session: {ctx.session}")
```
Comment (gautamgambhir97) [inline integrations/career-craft/src/agents/cold mailing/agent.py:49]: Remove commented-out code

**Review by gautamgambhir97 — COMMENTED** [review]
Overall, LGTM

**Review by cmaliwal — COMMENTED** [review]
LGTM

**Review by gautamgambhir97 — APPROVED** [review]

**Review by cmaliwal — APPROVED** [review]

File: integrations/career-craft/.env.sample [hunk 5]
```diff
@@ -0,0 +1 @@
+ResumeOptimizerPro_API_KEY=021e822e3amsh9b9670a5f742f31p1edc7jesn2a4ae8a6bb9f
```
[the remark this answers is not in the document]
Comment (TechyMT) [reply integrations/career-craft/.env.sample]: umm its not the actual key, it is a fake key

**Review by gautamgambhir97 — APPROVED** [review]

**Comment by TechyMT** [comment]
we are done with the changes

**Pull request merged**